framework/components/timelines/_timelines.scss

```diff
@@ -6,64 +6,96 @@ Markup: html/timeline.html
 Style guide timeline
 */
 
+/* Cross dependency from Buttons */
+@mixin whitespace-reset() {
+  line-height: 1.667;
+  padding-bottom: 0.5em;
+  padding-top: 0.5em;
+  white-space: normal;
+}
+
+%btn-ctn {
+  @include whitespace-reset();
+
+  background-color: $yellow-light;
+  color: $black;
+
+  &:hover,
+  &:active {
+    background-color: $yellow-light;
+    border-color: $text-color;
+  }
+
+  &:focus {
+    border-color: $text-color;
+    outline-color: $text-color;
+  }
+
+  &::after {
+    @extend %icon--after;
+    @extend %icon--right;
+
+    font-size: 0.8em;
+    margin-left: 10px;
```

Comment:
It is in timeline component, so it will be handled when refactoring it. I just edited the file to remove the cross dependency